assignments/homework/sebastian/HW 7 and 8.py

```diff
@@ -0,0 +1,65 @@
+#Notebook 7
+hdu_gaia = fits.open('data/gaia_lmc_psc.fits')
```

Comment:
The path to the data files should be relative from where this file is located. So change this to:
```suggestion
hdu_gaia = fits.open('../../../astropy_notebooks/data/gaia_lmc_psc.fits')
```